test/jdk/java/awt/Multiscreen/MultiScreenLocationTest/MultiScreenLocationTest.java

```diff
@@ -89,9 +92,9 @@ public static void main(String[] args) throws AWTException
             frame.setVisible(true);
             robot.waitForIdle();
             Rectangle bounds = frame.getBounds();
-            if (!Util.testBoundsColor(bounds, color, 5, 1000, robot)) {
-                throw new RuntimeException(getErrorText("Robot.getPixelColor", i));
-            }
+//            if (!Util.testBoundsColor(bounds, color, 5, 1000, robot)) {
+//                throw new RuntimeException(getErrorText("Robot.getPixelColor", i));
+//            }
```

Comment:
Why was this commented out? And if it's commented out, isn't it better to just be removed? It still seems to pass with it included. 

I tested with the newest changes. It passes on my local machine running macOS Monterey 12.3.1 using dual monitors. The test does create a yellow frame on both monitors and the color check passes in the test. I ran the test 5 times locally without any failures.

Reply:
Oops, this was for debugging purposes, let me remove this

Reply:
@DamonGuy can you double check that this portion of the test also passes on your machine?

Reply:
I checked again with and without `testBoundsColor` conditional, and both passed with a dual monitor setup on macOS Monterey 12.3.1